specification/theme/css/variables.css

```diff
@@ -0,0 +1,280 @@
+/* Globals */
```

Comment:
Deliberate?

Reply:
Yes, I widened the width of content in the ndc-spec documentation website by 150px to help give a bit more space to accommodate the capabilities tables